src/install/z_install.erl

```diff
@@ -69,6 +69,7 @@ install(Context) ->
     ok.
 
 
+-spec install_sql_list(#context{}, list()) -> ok.
```

Comment:
The spec in line 72 uses a record, please define a type for the record and use that instead.